### ISSUE TITLE
"Error reading 'o' stream in .frx data" after the first export

### ISSUE THREAD

**Issue by native-api** [chelh/VBASync#35]
* Export a form
* Reopen VBASync
* "Refresh"
* Error message box:

```
 ---------------------------
VBA Sync Tool
---------------------------
Error reading 'o' stream in .frx data: expected cbArrayString size 164, but saw 84 bytes as read.
---------------------------
ОК   
---------------------------
```

Maybe this has something to do with national characters in paths and/or in form properties.

This is a critical bug since the utility can't proceed any further.

**Comment by chelh** [comment]
Could you please attach a `.zip` file with your Office file and the contents of your repository folder, so I can reproduce?

**Comment by native-api** [comment]
Example file where the error is reproduced.

[А4_СУПЕР_СВИДЕТЕЛЬСТВО_97 - копия.zip](https://github.com/chelh/VBASync/files/1133300/4_._._97.-.zip)

Repository folder contents:
[VBAProjectFiles.zip](https://github.com/chelh/VBASync/files/1133302/VBAProjectFiles.zip)

**Comment by chelh** [comment]
Please replace your `VBASync.exe` and `VBASync.WPF.dll` with the corresponding files from this mid-development build, to see if you have a better experience. This should also help with #34.

– Disregard this version, I will send a new one shortly

After extracting, right-click the `.exe` and the `.dll` separately, and for each go into properties and hit **Unblock** (or the Russian equivalent, it should be near the bottom-right) if it appears.

**Comment by chelh** [comment]
I've updated the mid-development build with a fix specifically for #34 (I thought it had been fixed by an earlier commit, but I guess not!):

[Portable VBA Sync Tool 2.2.0.zip](https://github.com/chelh/VBASync/files/1133523/Portable.VBA.Sync.Tool.2.2.0.zip)

**Comment by native-api** [comment]
This bug is fixed!
#34, however, is not.

P.S. thanks for the prompt response. I got in hospital and couldn't check the fix out earlier.
